glide.lock

```diff
@@ -149,7 +149,7 @@ imports:
   subpackages:
   - go
 - name: github.com/prometheus/common
-  version: 195bde7883f7c39ea62b0d92ab7359b5327065cb
+  version: 9e0844febd9e2856f839c9cb974fbd676d1755a8
```

Comment:
Seems we're not pinning this ourselves and its being brought in transitively through our dependency on `github.com/prometheus/client_model`.